bin/orm_exporttopostgres

```diff
@@ -0,0 +1,341 @@
+#!/usr/bin/env python3
+
+# Script to import the XML based pint data into a PostgreSQL DB
+# THis is a one time operation, the database is expceted to hav a table called
+# "postgres" that has no entries
+
+import argparse
+from datetime import datetime
+import glob
+import os
+import subprocess
+import sys
+
+from lxml import etree
+
+from pint_server.database import init_db
+from pint_server.models import (
+            ImageState,
+            ServerType,
+            AlibabaImagesModel,
+            AmazonImagesModel,
+            AmazonServersModel,
+            GoogleImagesModel,
+            GoogleServersModel,
+            MicrosoftImagesModel,
+            MicrosoftRegionMapModel,
+            MicrosoftServersModel,
+            OracleImagesModel,
+            VersionsModel
+        )
+
+import logging
+
+
+def gen_data_files_list(pint_data_repo=None):
+    
+    if pint_data_repo is None:
+        pint_data_repo = os.path.realpath(
+                             os.path.join(
+                                 os.path.dirname(os.path.abspath(__file__)),
+                                 "..", "..", "pint-data"))
+
+    logger.debug("Pint Data repo: %s", repr(pint_data_repo))
+
+    data_files_pattern = os.path.join(pint_data_repo, 'data', '*.xml')
+    logger.debug("Data Files pattern: %s", repr(data_files_pattern))
+
+    data_files = glob.glob(data_files_pattern)
+    logger.debug("Data Files: %s", repr(data_files))
+
+    return data_files
+
+def get_commit_date(data_file):
```

Comment:
What if we have multiple commits in a day? According to Robert, that's a possibility.

Reply:
So using the commit time stamp info isn't perfect in and of itself anyway, as you can merge commits out of order, so a change that was proposed yesterday could be merged after one that was proposed this morning, and you end up with the version date being yesterday.

I used the commit date instead of just the runtime date stamp to allow me to track the relationship between the incremental merges as I made them in testing...

However we could switch back to the original model of using the timestamp from when the command is run.

Reply:
I believe I have addressed this issue in my latest change, as well as removing some redundant code, and fixing a bug related to not using the correct table name when doing the existing version query.

Basically we will take the "suggested" version, and if it is newer than the existing entry, we will use it, otherwise we will increment the existing value by 0.01, which will handle older commit date stamps being used due to out of order merges as well as up to 100 incremental updates per day while still maintaining the logical relationship between commit date stamp and version, and will still support >100 incremental updates per-day with that logical relationship being lost due to the version value "rolling over" to the next logical day...